app/views/shared/_upcoming_events.html.erb

```diff
@@ -0,0 +1,44 @@
+<section>
+    <h1> Upcoming Events </h1>
+    <h3> See What's Next! </h3>
+    <div class="container marketing">
```

Comment:
I'm assuming that the other file will be deleted. I'm not sure what this 'marketing' class is doing. If it isn't being used for anything please remove it.